out/main.go

```diff
@@ -60,7 +65,7 @@ func main() {
 		Number: v.String(),
 	}
 
-	err = bucket.Put(request.Source.Key, []byte(outVersion.Number), "text/plain", "")
+	err = bucket.Put(request.Source.Key, []byte(outVersion.Number), "text/plain", s3.BucketOwnerFull)
```

Comment:
My bad, changed to to private now.
I just had to pick something as my S3 provider can't handle empty values.